openapi3.yaml

```diff
@@ -1,41 +1,34 @@
 openapi: 3.0.1
 info:
-  title: service-name
-  description: basic template for map colonies service
+  title: external-to-geojson
+  description: 'convert external format to geoJson format'
   version: 1.0.0
   license:
     name: MIT
     url: https://opensource.org/licenses/MIT
 servers:
   - url: http://localhost:8080
 paths:
-  /resourceName:
-    get:
-      operationId: getResourceName
+  /convert:
+    post:
       tags:
-        - resourceName
-      summary: gets the resource
+        - convert
+      summary: convert body record to geoJson
       responses:
         200:
-          description: OK
+          description: converted
```

Comment:
I think the URL should be POST `/sources/{external_source_name}/convert`

even though we are currently working on one external source, there is no stopping us from generalizing the service for more scehmas

Reply:
Done